docs/fundamentals/networking/http/httpclient-migrate-from-httpwebrequest.md

```diff
@@ -0,0 +1,136 @@
+await stream.WriteAsync("Hello World!"u8.ToArray());
+using WebResponse response = await request.GetResponseAsync();
+Memory<byte> buffer = new byte[1024];
+await using Stream responseStream = response.GetResponseStream();
+```
+
+#### Simple POST Request Using <xref:System.Net.Http.HttpClient>
+
+Here's an example of how the code might look:
+
+```c#
+using System.Net.Http;
+
+using HttpClient client = new();
+using HttpResponseMessage responseMessage = await client.PostAsync(uri, new StringContent("Hello World!"));
+```
+
+## 1:1 API Mapping Table
+
+Developers should be aware that `ServicePointManager` is a static class, meaning that any changes made to its properties will have a global effect on all newly created `ServicePoint` objects within the application. For example, modifying a property like `ConnectionLimit` or `Expect100Continue` will impact every new ServicePoint instance.
+
+### <xref:System.Net.ServicePointManager> Properties Mapping
+
+| <xref:System.Net.ServicePointManager> Old API | New API | Notes |
+|---------|----------------------|-------|
+| `CheckCertificateRevocationList` | <xref:System.Net.Http.SocketsHttpHandler.SslOptions>.CertificateRevocationCheckMode | See TODO: for usage |
+| `DefaultConnectionLimit` | <xref:System.Net.Http.SocketsHttpHandler.MaxConnectionsPerServer> | TODO |
+| `DnsRefreshTimeout` | No equivalent API | Custom implementation needed |
+| `EnableDnsRoundRobin` | No equivalent API | Custom implementation needed |
+| `EncryptionPolicy` | No equivalent API | TODO |
+| `Expect100Continue` | <xref:System.Net.Http.Headers.HttpRequestHeaders.ExpectContinue> | TODO |
+| `MaxServicePointIdleTime` | <xref:System.Net.Http.SocketsHttpHandler.PooledConnectionIdleTimeout> | TODO |
+| `MaxServicePoints` | No equivalent API | TODO |
+| `ReusePort` | No direct equivalent API | Can be done over <xref:System.Net.Http.SocketsHttpHandler.ConnectCallback>. |
+| `SecurityProtocol` | <xref:System.Net.Http.SocketsHttpHandler.SslOptions>.EnabledSslProtocols | TODO |
+| `ServerCertificateValidationCallback` | <xref:System.Net.Http.SocketsHttpHandler.SslOptions>.RemoteCertificateValidationCallback | Signatures are different need a custom callback |
+| `UseNagleAlgorithm` | No direct equivalent API | Can be done over <xref:System.Net.Http.SocketsHttpHandler.ConnectCallback>. |
+
+### <xref:System.Net.ServicePointManager> Method Mapping
+
+| <xref:System.Net.ServicePointManager> Old API | New API | Notes |
+|---------|----------------------|-------|
+| `FindServicePoint` | No equivalent API | TODO |
+| `SetTcpKeepAlive` | No direct equivalent API | Can be done over <xref:System.Net.Http.SocketsHttpHandler.ConnectCallback>. |
+
+### <xref:System.Net.ServicePoint> Properties Mapping
+
+| <xref:System.Net.ServicePoint> Old API | New API | Notes |
+|---------|----------------------|-------|
+| `Address` | `HttpRequestMessage.RequestUri` | This is request uri, so basically we can get this from `HttpRequestMessage`. |
```

Comment:
I think the ", so basically" filler words should be avoided, same for use of first person plural (we can....)